docs/docs/explore/project-chat.md

```diff
@@ -0,0 +1,169 @@
+
+### Visual Components
+
+Unlike text-only AI assistants, AI Chat in Rill Cloud can render actual dashboard components directly in the chat interface. These visualizations are built using the same [canvas dashboard](/build/dashboards/canvas-widgets) technology used throughout Rill, ensuring:
+
+- **Consistency** - Visualizations match the style and behavior of your regular dashboards
+- **Interactivity** - Click, hover, and interact with charts right in the chat
+- **Accuracy** - Charts are generated from the same data sources as your dashboards
+- **Clarity** - Complex data patterns become immediately visible
+
+:::tip From Chat to Dashboard
+Think of AI Chat as your starting point for data exploration. Use it to quickly answer questions and spot trends, then leverage the dashboard links to dive deeper with full filtering, comparison, and export capabilities.
+:::
+
+## Improving AI Chat with Instructions
+
+To get the most accurate and contextual responses from AI Chat, you can add custom `ai_instructions` to your project files. These instructions provide the AI with additional context about your data, business logic, and preferred response formats.
+
+### Why Add AI Instructions?
+
+AI instructions help the AI:
+- Understand your specific business context and terminology
+- Format responses in ways that match your team's preferences
+- Generate properly formatted Explore dashboard links
+- Focus on the metrics and dimensions most relevant to your use case
+- Provide more actionable insights tailored to your workflows
+
+### Where to Add Instructions
+
+You can add `ai_instructions` in two places:
+
+1. **`rill.yaml`** - Project-wide instructions that apply to all queries across your entire project
+2. **`<metrics_view>.yaml`** - Metrics view-specific instructions for individual dashboards
+
+For detailed examples and best practices on writing effective AI instructions, see the [Rill MCP documentation](/explore/mcp#adding-ai-instructions-to-your-model).
+
+
+## AI Chat vs. Rill MCP Server
+
+Rill offers two ways to chat with your data using AI:
+
+| Feature      | AI Chat (in Rill Cloud)                    | Rill MCP Server                               |
+| ------------ | ------------------------------------------ | --------------------------------------------- |
+| **Location** | Built into Rill Cloud browser interface    | External AI assistants (Claude Desktop, etc.) |
+| **Setup**    | No setup required                          | Requires MCP client configuration             |
+| **Access**   | Any Rill Cloud user with project access    | Requires personal access token                |
+| **Use Case** | Quick questions while exploring dashboards | Deep analysis sessions in your AI assistant   |
+| **Best For** | Business users and analysts                | Data teams and power users                    |
+
+Both use the same underlying technology and provide equally accurate results based on your metrics views.
```

Comment:
I'm wondering if we've nailed the delineation between the AI Chat and the MCP pages. First, placing AI Chat before MCP will help. Then, mind reviewing the transitions, links, and any redundancies between the two pages?